_posts/2019-08-26-how-to-use-gruntwork-infrastructure-as-code-library.adoc

```diff
@@ -640,140 +640,124 @@ next two sections of the guide will walk you through how to use each of these.
-  variable (you'll declare this shortly). This is an extra safety measure to ensure you don't accidentally authenticate
-  to the wrong AWS account while deploying this code—e.g., so you don't accidentally deploy changes intended for
-  staging to production (for more info on working with multiple AWS accounts, see
-  link:/guides/foundations/how-to-configure-production-grade-aws-account-structure[How to Configure a Production Grade AWS Account Structure]).
+Pin the Terraform version::
+  The code above will ONLY allow you to run it with a specific Terraform version. This is a safety measure to ensure
+  you don't accidentally pick up a new version of Terraform until you're ready. This is important because (a) Terraform
+  is a pre 1.0.0 tool, so even patch version number bumps (e.g., `0.12.6` -> `0.12.7`) are sometimes backwards
+  incompatible or buggy and (b) once you've upgraded to a newer version, Terraform will no longer allow you to deploy
+  that code with any older version. For example, if a single person on your team upgrades to `0.12.7` and runs `apply`,
+  then you'll no longer be able to use the state file with `0.12.6`, and you'll be forced to upgrade everyone on your
+  team and all your CI servers to `0.12.7`. It's best to do this explicitly, rather than accidentally, so we recommend
+  pinning Terraform versions.
 
-Let's add the corresponding input variables in `variables.tf`:
 
-.infrastructure-modules/networking/vpc-app/variables.tf
+Let's look at the corresponding input variables in `variables.tf`. You'll provide values for these when you start using the
+module in the next section.
+
+.terraform-aws-service-catalog/modules/networking/vpc/variables.tf
 [source,hcl]
 ----
 variable "aws_region" {
   description = "The AWS region in which all resources will be created"
   type        = string
 }
 
-variable "aws_account_id" {
-  description = "The ID of the AWS Account in which to create resources."
+variable "vpc_name" {
+  description = "Name of the VPC. Examples include 'prod', 'dev', 'mgmt', etc."
   type        = string
 }
-----
-
-==== Configure Terraform
-
-Next, configure Terraform itself in `main.tf`:
 
-.infrastructure-modules/networking/vpc-app/main.tf
-[source,hcl]
-----
-terraform {
-  # Partial configuration for the backend: https://www.terraform.io/docs/backends/config.html#partial-configuration
-  backend "s3" {}
-
-  # Only allow this Terraform version. Note that if you upgrade to a newer version, Terraform won't allow you to use an
-  # older version, so when you upgrade, you should upgrade everyone on your team and your CI servers all at once.
-  required_version = "= 0.12.6"
+variable "cidr_block" {
+  description = "The IP address range of the VPC in CIDR notation. A prefix of /18 is recommended. Do not use a prefix higher than /27. Examples include '10.100.0.0/18', '10.200.0.0/18', etc."
+  type        = string
 }
-----
```

Comment:
NOTE: This section is still relevant for calling service catalog module with pure terraform. The Service Catalog modules do not configure a backend.